chajda/tsquery/augments.py

```diff
@@ -52,62 +52,185 @@ def augments_gensim(lang, word, config=Config(), n=5):
 
     return words
 
-
+import os
 import fasttext
 import fasttext.util
+from annoy import AnnoyIndex
+
+
+chajda_path = os.path.dirname(__file__)
+working_dir = os.getcwd()
```

Comment:
this has a bug in it; what if the cwd changes between when this line is executed and when the other functions are called?  then when you reset the cwd later, you'll be setting it to the old path

you can fix this bug by moving this line into the function